extractor/main.go

```diff
@@ -105,10 +105,23 @@ func execExtract(ctx context.Context, cfg *extractorCfg) error {
 		return errInvalidOutputDir
 	}
 
-	// Find the files that need to be analyzed
-	sourceFiles, findErr := findFilePaths(cfg.sourceDir, cfg.fileType)
-	if findErr != nil {
-		return fmt.Errorf("unable to find file paths, %w", findErr)
+	var sourceFiles []string
+	var findErr error
+
+	// Check if source is valid*
+	source, err := os.Stat(cfg.sourcePath)
+	if err != nil {
+		return fmt.Errorf("unable to open source, %w", err)
```

Comment:
The error should be `unable to stat source path`

Reply:
Updated the error: [1d28df6](https://github.com/gnolang/tx-exports/pull/11/commits/1d28df6018edbad4b9a9701aa89e316aae473fdb)